SettingsActivityTest fails due to undeclared dependencies

**Summary:** 

SettingsActivityTest fails due to undeclared dependencies. This happens locally and on Travis-CI.

**Steps to reproduce:** 

Run `gradle :app:testBetaDebugUnitTest` with an emulated/real device connected.

It will give:
```
> Task :app:compileBetaDebugAndroidTestJavaWithJavac
/home/whym/projects/commons-app/app/src/androidTest/java/fr/free/nrw/commons/SettingsActivityTest.java:11: error: package android.support.test.rule does not exist
import android.support.test.rule.ActivityTestRule;
                                ^
/home/whym/projects/commons-app/app/src/androidTest/java/fr/free/nrw/commons/SettingsActivityTest.java:30: error: cannot find symbol
    public ActivityTestRule<SettingsActivity> activityRule =
           ^
  symbol:   class ActivityTestRule
  location: class SettingsActivityTest
/home/whym/projects/commons-app/app/src/androidTest/java/fr/free/nrw/commons/SettingsActivityTest.java:31: error: cannot find symbol
            new ActivityTestRule<SettingsActivity>(SettingsActivity.class,
                ^
  symbol:   class ActivityTestRule
  location: class SettingsActivityTest
3 errors
```

**Expected behavior:** 

There should be no error in the test log.

**Would you like to work on the issue?**

Yes (assigned).

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -75,6 +75,8 @@ dependencies {
     androidTestImplementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
     androidTestImplementation 'com.squareup.okhttp3:mockwebserver:3.8.1'
     androidTestImplementation "com.android.support:support-annotations:$SUPPORT_LIB_VERSION"
+    androidTestImplementation 'com.android.support.test:rules:1.0.2'
+    androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 
     debugImplementation "com.squareup.leakcanary:leakcanary-android:$LEAK_CANARY"

